File: C++_Programming/Practice/program9.cpp
/*
    START
        Accept number as No
        if No is completely divisible by 2
            then print Even
        otherwise
            print odd
    STOP 
    
    
    START
        Accept number as No
        divide No by 2
        If remainder is 0 
            then print as Even
        otherwise
            print as odd
        STOP
*/

#include<iostream>
using namespace std;

int main()
{   
    int iValue = 0;
    int iRemainder = 0;

    cout<<"Enter Number : ";
    cin>>iValue;

    iRemainder = iValue % 2;

    if(iRemainder == 0)
    {
        cout<<"The Number is Even";
    }
    else
    {
        cout<<"The Number is Odd";
    }

    return 0;
}

File: Java_Programming/Practice/program9.java
import java.util.Scanner;

class program9
{
    public static void main(String A[]) 
    {
        Scanner sobj = new Scanner(System.in);
        int iValue = 0;
        int iRemainder = 0;

        System.out.println("Enter Number : ");
        iValue = sobj.nextInt();

        iRemainder = iValue % 2;

        if(iRemainder == 0)
        {
            System.out.println("Number Is Even");
        }
        else
        {
            System.out.println("Number Is Odd");
        }
    }
}